PHPCI/Controller/BuildController.php

```diff
@@ -198,11 +199,8 @@ public function delete($buildId)
     */
     protected function cleanLog($log)
     {
-        $log = str_replace('[0;32m', '<span style="color: green">', $log);
```

Comment:
I think this is a side-effect of PHPCI running a build for each commit in a pull request. Going to revise that behaviour. :)